packages/edit-site/src/components/left-sidebar/navigation-panel/index.js

```diff
@@ -0,0 +1,186 @@
+/**
+ * WordPress dependencies
+ */
+import { useState } from '@wordpress/element';
+import {
+	__experimentalNavigation as Navigation,
+	__experimentalNavigationGroup as NavigationGroup,
+	__experimentalNavigationItem as NavigationItem,
+	__experimentalNavigationMenu as NavigationMenu,
+} from '@wordpress/components';
+import { getBlockType, getBlockFromExample } from '@wordpress/blocks';
+import { BlockPreview } from '@wordpress/block-editor';
+
+const NavigationPanel = () => {
+	const [ activeItem, setActiveItem ] = useState( 'item-1' );
+	const [ activeMenu, setActiveMenu ] = useState( 'root' );
```

Comment:
The `activeMenu` state is optional, only useful if we need to control the hierarchical navigation from outside.
It's not really needed here at this point.